src/Processors/Transforms/Streaming/AggregatingTransform.cpp

```diff
@@ -549,6 +549,10 @@ void AggregatingTransform::recover(CheckpointContextPtr ckpt_ctx)
             {
                 UInt64 last_rows = 0;
                 DB::readIntBinary<UInt64>(last_rows, rb);
+                /// In case when for we had global aggregated some data, but done checkpoint request before finializing
```

Comment:
I feel like we don't need this logging and we will need fix the crash which is the significant part. With this logging, we can do nothing ?